### PR TITLE
OpenPaaS-Suite/esn-frontend-contacts#33

### LINKED ISSUE #33: Unable to subscribe to shared address book
Steps to reproduce:
1. Go to https://dev.open-paas.org/contacts/
2. Click (+) button on the left menu and select "Shared address book" option

Expected result:
"Shared address book" popup is displayed and user can search for other users and select address books to subscribe

Actual result:

There is no field to search for user 

Linked PR: https://github.com/OpenPaaS-Suite/esn-frontend-common-libs/pull/100

### PR DESCRIPTION
**Pull request by piavgh**
Partially resolves OpenPaaS-Suite/esn-frontend-contacts#33

Leftover:
- [x] Fix placeholder from 'Add a tag' to 'Users'

### CODE REVIEW

File: src/frontend/js/modules/user/users-autocomplete-input/users-autocomplete-input.directive.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 'use strict';
 
+const _ = require('lodash');
```
Comment (tuanlc) [inline src/frontend/js/modules/user/users-autocomplete-input/users-autocomplete-input.directive.js:3]: 1.  Even we do not have eslint now, but it is better if we keep our convention by adding 1 blank line after defining vars
2.  The commit message `Replace lodash Import ` is not good IMO.

File: src/frontend/views/modules/user/users-autocomplete-input.pug [hunk 2]
```diff
@@ -1,3 +1,3 @@
 .col-xs-12
-  tags-input.form-control(ng-model='mutableUsers', add-on-enter='{{!propagateEnterEvent}}', display-property='displayName', key-property='id', placeholder='{{ translatedPlaceholder }}', on-tag-adding='onAddingUser($tag)', on-tag-added='onUserAdded($tag)', on-tag-removed='onUserRemoved($tag)', add-from-autocomplete-only='{{::addFromAutocompleteOnly}}', replace-spaces-with-dashes='false')
+  tags-input.form-control(ng-model='mutableUsers', add-on-enter='{{!propagateEnterEvent}}', display-property='displayName', key-property='id', placeholder='{{ translatedPlaceholder || "Users" }}', on-tag-adding='onAddingUser($tag)', on-tag-added='onUserAdded($tag)', on-tag-removed='onUserRemoved($tag)', add-from-autocomplete-only='{{::addFromAutocompleteOnly}}', replace-spaces-with-dashes='false')
```
Comment (tuanlc) [inline src/frontend/views/modules/user/users-autocomplete-input.pug]: By this, you do not have the translation for `Users` in other locales

Comment (piavgh) [reply src/frontend/views/modules/user/users-autocomplete-input.pug]: Fixed by you :trollface: Thanks

File: src/frontend/js/modules/user/users-autocomplete-input/users-autocomplete-input.directive.js (hunk 1 above)
Comment (piavgh) [reply src/frontend/js/modules/user/users-autocomplete-input/users-autocomplete-input.directive.js:3]: Fixed it

**Review by tuanlc — APPROVED** [review]

**Review by itswillta — APPROVED** [review]

**Pull request merged**